modules/core/src/main/scala/scala/build/errors/AmbiguousPlatformError.scala

```diff
@@ -0,0 +1,5 @@
+package scala.build.errors
+
+final class AmbiguousPlatformError extends BuildException(
+      "Ambiguous platform: more than one type of platform has been passed."
```

Comment:
optimally, we could check what particular types of platforms were passed and then mention them in the error, so that the user knows what to look for in his implementation.
```suggestion
      s"Ambiguous platform: more than one type of platform has been passed: ${passedPlatformTypes.mkString(", "}."
```